docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -97,6 +97,35 @@ You can also use `winget`, Microsoft's Package manager for Windows, to install a
 
 The `-e` option is to ensure the official Azure CLI package is installed. This command installs the latest version by default. To specify a version, add a `-v <version>` with your desired version to the command.
 
+# [ZIP Package](#tab/zip)
+
+You can use the ZIP package to install the Azure CLI on Windows. This package is useful when you don't have administrative privilege.
+
+Please unzip the package to a folder, then you can run the Azure CLI with the `<unzipped folder path>\wbin\az.cmd` command. (If your path contains spaces, please use `& "<unzipped folder path>\wbin\az.cmd"`)
+
+If you want to run the `az` command directly, then follow the steps below to add the Azure CLI to your PATH environment variable.
+1. Open the Start Menu and search for `environment variables`, click `Edit the system environment variables`.
+2. Click `Environment Variables...` button.
```

Comment:
You can also use `settings -> system -> about ->Advanced system settings` to open this dialog, although it requires more steps.